pkgs/development/php-packages/zstd/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPecl, lib, zstd, pkg-config, fetchFromGitHub }:
+
+let
+  version = "0.13.3";
+in buildPecl {
+  inherit version;
+  pname = "zstd";
+
+  src = fetchFromGitHub {
+    owner = "kjdev";
+    repo = "php-ext-zstd";
+    rev = version;
+    sha256 = "sha256-jEuL93ScF0/FlfUvib6uZafOkIe0+VkWV/frpSjTkvY=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.